drivers/scheduler/k8s/k8s.go

```diff
@@ -990,7 +990,7 @@ func (k *k8s) InspectVolumes(ctx *scheduler.Context) error {
 				}
 			}
 
-			if err := k8sOps.ValidatePVCsForStatefulSet(ss); err != nil {
+			if err := k8sOps.ValidatePVCsForStatefulSet(ss, timeout*3, retryInterval); err != nil {
```

Comment:
You cannot hardcoded 3 here. You need to lookup in the `ss` for it's replica count. Ref: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#statefulsetspec-v1-apps

Reply:
Yes as it takes that much time for the entire ss to come up and all of it's PVCs to be bound.